diff --git a/xmlsecurity/Module_xmlsecurity.mk b/xmlsecurity/Module_xmlsecurity.mk
--- a/xmlsecurity/Module_xmlsecurity.mk
+++ b/xmlsecurity/Module_xmlsecurity.mk
@@ -11,7 +11,7 @@ $(eval $(call gb_Module_Module,xmlsecurity))
 $(eval $(call gb_Module_add_targets,xmlsecurity,\
 	Library_xmlsecurity \
 	$(if $(ENABLE_NSS)$(ENABLE_OPENSSL),Library_xsec_xmlsec) \
-	UIConfig_xmlsec \
+	$(if $(DISABLE_GUI),,UIConfig_xmlsec) \
 ))
 
 $(eval $(call gb_Library_use_custom_headers,xmlsecurity,\
@@ -29,7 +29,7 @@ $(eval $(call gb_Module_add_subsequentcheck_targets,xmlsecurity,\
 ))
 
 $(eval $(call gb_Module_add_l10n_targets,xmlsecurity,\
-	AllLangMoTarget_xsc \
+	$(if $(DISABLE_GUI),,AllLangMoTarget_xsc) \
 ))
 
 # failing
