diff --git a/svx/util/svxcore.component b/svx/util/svxcore.component
--- a/svx/util/svxcore.component
+++ b/svx/util/svxcore.component
@@ -25,7 +25,6 @@
   <implementation name="com.sun.star.comp.Draw.GraphicExporter"
       constructor="com_sun_star_comp_Draw_GraphicExporter_get_implementation">
     <service name="com.sun.star.drawing.GraphicExportFilter"/>
-    <optional/>
   </implementation>
   <implementation name="com.sun.star.comp.svx.Impl.FindbarDispatcher"
       constructor="com_sun_star_comp_svx_Impl_FindbarDispatcher_get_implementation">
diff --git a/svx/util/svxcore.component.draw b/svx/util/svxcore.component.draw
--- a/svx/util/svxcore.component.draw
+++ b/svx/util/svxcore.component.draw
@@ -1,7 +1,7 @@
 # This file is part of the LibreOffice project.
 #
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-com.sun.star.comp.Draw.GraphicExporter
+

