diff --git a/solenv/gbuild/platform/EMSCRIPTEN_INTEL_GCC.mk b/solenv/gbuild/platform/EMSCRIPTEN_INTEL_GCC.mk
index df7503a50..db871647c 100644
--- a/solenv/gbuild/platform/EMSCRIPTEN_INTEL_GCC.mk
+++ b/solenv/gbuild/platform/EMSCRIPTEN_INTEL_GCC.mk
@@ -30,8 +30,8 @@ gb_EMSCRIPTEN_QTDEFS := -DQT_NO_LINKED_LIST -DQT_NO_JAVA_STYLE_ITERATORS -DQT_NO
 gb_Executable_EXT := .html
 gb_EMSCRIPTEN_EXCEPT = -fwasm-exceptions -s SUPPORT_LONGJMP=wasm
 
-# Exported functions for the LibreOfficeKit API
-gb_EMSCRIPTEN_LDFLAGS += -s EXPORTED_FUNCTIONS=["_lok_documentLoad","_lok_documentLoadWithOptions","_lok_documentSaveAs","_lok_documentDestroy","_lok_destroy","_lok_getError","_libreofficekit_hook","_libreofficekit_hook_2","_lok_preinit","_lok_preinit_2","_main","_malloc","_free"]
+# EXPORTED_FUNCTIONS for LOK API moved to desktop/Executable_soffice_bin.mk
+# They should only apply to the final soffice link, not to external libraries
 
 gb_CXXFLAGS += $(gb_EMSCRIPTEN_CPPFLAGS)
 
